# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/midi/MIDIPort.h
# Commit: cfba299e5067
# Full Hash: cfba299e5067a4eab3f3b982374b0770970a3401
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-04-12 03:57:01
# Regressor Bug: 1758468
# File Overlap Count: 1
# Description:
#   Bug 1758468 - Use stable, anonymous IDs for Web MIDI ports r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142550
# ==============================================================================

diff -r c0eb987d0f76 -r cfba299e5067 dom/midi/MIDIPort.h
--- a/dom/midi/MIDIPort.h	Mon Apr 11 17:43:08 2022 +0000
+++ b/dom/midi/MIDIPort.h	Mon Apr 11 17:43:09 2022 +0000
@@ -73,6 +73,7 @@
   IMPL_EVENT_HANDLER(statechange)
 
   void DisconnectFromOwner() override;
+  const nsString& StableId();
 
  protected:
   // IPC Actor corresponding to this class